### PR TITLE
Add Minesweeper example application

### PR DESCRIPTION
**Pull request by VeselovAlex**

### CODE REVIEW

File: examples/minesweeper/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Minesweeper
+
+A simple game powered by Jetpack Compose and Kotlin Multiplatform
```
Comment (olonho) [inline examples/minesweeper/README.md:3]: Compose Multiplatform

File: examples/minesweeper/settings.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+pluginManagement {
```
Comment (olonho) [inline examples/minesweeper/settings.gradle.kts:1]: Maybe no longer needed.

**Review by olonho — APPROVED** [review]

**Review by olonho — CHANGES_REQUESTED** [review]
Please migrate to 1.0.0-rc6 and get rid of warnings.

**Review by olonho — APPROVED** [review]

File: examples/minesweeper/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
+import org.jetbrains.compose.compose
+
+plugins {
+    kotlin("multiplatform") version "1.5.31"
+    id("org.jetbrains.compose") version "1.0.0-rc6"
```
Comment (igordmn) [inline examples/minesweeper/build.gradle.kts:6]: - use 1.0.0
- migrate to new template:
https://github.com/JetBrains/compose-jb/commit/dbd0a21949633302fcf98ad79c78896bbc6250ba#diff-7d46308093aec3efc8dd6079383a7b856efa79af87787a1fb2898504e0c4d027
image
-  add this example to https://github.com/JetBrains/compose-jb/blob/master/examples/validateExamples.sh

**Comment by olonho** [comment]
Looks good, but not yet native targets ready.

**Pull request merged**